src/app/api/v1/recordings/route.ts

```
@@ -0,0 +1,42 @@
+import { APIError } from '@/global/exceptions';
+import { IRecording, IRecordingListingResponse } from '@/global/types';
+import MESSAGES from '@/global/messages';
+import { API_ERROR_CODE } from '@/global/constants/errorCodes';
+import { LISTING_LIMIT } from '@/global/constants/listingLimit';
+import getReqBody from '../../api-utils/getReqBody';
+import { recordingsCollRef } from '../firestoreRefs';
+
+export const POST = withErrorHandling(async (req: NextRequest) => {
+	const { page = 1, limit = LISTING_LIMIT } = await getReqBody(req);
+
+	if (!page || isNaN(page) || Number(page) < 1 || isNaN(limit) || Number(limit) < 1) throw new APIError(`${MESSAGES.REQ_BODY_ERROR}`, 500, API_ERROR_CODE.REQ_BODY_ERROR);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)